radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/TimeoutCertificate.java

```diff
@@ -99,8 +99,8 @@
 	@JsonCreator
 	private static TimeoutCertificate serializerCreate(
 		@JsonProperty("epoch") long epoch,
```

Comment:
Same here about negative epoch and view values. Should encapsulate them in an object?

Reply:
eh, would rather not have object encapsulation around epoch. It's awkward enough with the way we use `View` but a check out be good.

Reply:
added check